src/components/UserAccounts/AccountItem.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts" setup>
+import type { Account } from '/@/lib/apis'
+import ServiceLogo from '/@/components/UserAccounts/ServiceLogo.vue'
+import Icon from '/@/components/UI/Icon.vue'
+
+interface Props {
+  account: Account
+}
+
+const props = defineProps<Props>()
+</script>
+
+<template>
+  <router-link
+    :to="`/user/accounts/${props.account.id}/edit`"
+    :class="$style.link"
+  >
+    <div :class="$style.container">
+      <div :class="$style.displayNameContainer">
+        <p :class="$style.displayName">{{ props.account.displayName }}</p>
```

Comment:
```suggestion
        <p :class="$style.displayName">{{ account.displayName }}</p>
```
`props`は最初から参照できるので、`props.`つけなくても大丈夫です 👌